Gemfile.lock

```diff
@@ -56,25 +56,27 @@ GEM
     nap (1.1.0)
     netrc (0.11.0)
     rouge (2.0.7)
-    ruby-macho (1.2.0)
+    ruby-macho (1.4.0)
     thread_safe (0.3.6)
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    xcodeproj (1.6.0)
+    xcodeproj (1.8.2)
       CFPropertyList (>= 2.3.3, < 4.0)
       atomos (~> 0.1.3)
       claide (>= 1.0.2, < 2.0)
       colored2 (~> 3.1)
       nanaimo (~> 0.2.6)
     xcpretty (0.3.0)
       rouge (~> 2.0.7)
+    xcpretty-json-formatter (0.1.1)
+      xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods (~> 1.6.0.beta.1)
-  xcpretty
+  cocoapods (~> 1.7.0.beta.3)
+  xcpretty-json-formatter (~> 0.1.0)
```

Comment:
Should be `xcpretty`.

Reply:
Set back to xcpretty and re-ran bundle install